coral/src/app/features/topics/acl-request/TopicAclRequest.tsx

```diff
@@ -110,7 +110,7 @@ const TopicAclRequest = () => {
   const currentTopicNames = selectedEnvironment?.topicNames || [];
 
   return (
-    <Box maxWidth={"4xl"}>
+    <Box style={{ maxWidth: 1200 }}>
```

Comment:
What motivated this choice? :o Why use a hardcoded value rather than the standard ones provided by the component's API?

edit: ah, I'm guessing because `maxWidth={"4xl"}` is actually the outlier, and other spots use `1200` hahaha! However, I think the standardization should favour the native widths provided by the DS, as opposed to custom styles :o

Reply:
Yes, I used the one that was used more often 😅   and you're right, standardisation should be preferred! Will update.

Reply:
Yes! That's good. Also for whatever reason I remembered it being `em` instead of `rem`. Will update! Maybe that was a leftover from `Box` not having `maxWidth` and then me copy and pasting without 🧠  too much.